src/main.js

```diff
@@ -96,10 +96,31 @@ export default class LevelSelector extends UICorePlugin {
   configureLevelsLabels() {
     if (this.core.options.levelSelectorConfig === undefined) return
 
-    for (var levelId in (this.core.options.levelSelectorConfig.labels || {})) {
-      levelId = parseInt(levelId, 10)
-      var thereIsLevel = !!this.findLevelBy(levelId)
-      thereIsLevel && this.changeLevelLabelBy(levelId, this.core.options.levelSelectorConfig.labels[levelId])
+    var labelCallback = this.core.options.levelSelectorConfig.labelCallback
+    if(labelCallback && typeof labelCallback !== 'function')
```

Comment:
We usually follow the code convention of using the open brackets in the same line `if (x) {` (we need to enforce it in the future by using eslint (like Clappr did))